jumpscale/core/executors/tmux.py

```diff
@@ -5,37 +5,55 @@
 
 server = libtmux.Server()
 JS_SESSION_NAME = "js-ng"
+JS_WINDOW_NAME = "js-ng-w"
```

Comment:
hmm, not sure if i get it, but the idea is having a session for js-ng where u create multiple windows in there when you execute commands, but with this change you will be always executing your commands in a window named `js-ng-w` ?